src/components/Character/Character.jsx

```diff
@@ -15,38 +18,47 @@ export default function Character(props){
 
 	//RENDER
 	//----------------------------
+	const RENDER = { ...RENDER_A, ...RENDER_B };
```

Comment:
```suggestion
	const RENDER = { ...CHARACTER_RENDER, ...SHARED_RENDER };
```
Just to make it somewhat more useful in terms of naming